lib/github-release.rb

```diff
@@ -128,8 +133,15 @@ def git_config(item, default = nil)
 	end
 
 	def create_release(tag, prerelease)
-		print "Pushing to #{remote_name}...\n"
-		system("git push #{remote_name} tag #{tag}")
+		$pushed ||= false
```

Comment:
Your newness to Ruby didn't show particularly, if that helps.  :grinning: 

A variable prefixed by an @ is an "instance variable", and works equivalently to the Java or C++ equivalents -- it exists on a per-object basis, for the lifetime of that object.  Since the `GithubRelease` object, when run via the command line, exists for the lifetime of the script, an instance variable is the correct way to go.